lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -2483,6 +2550,14 @@ instance LiftHandler ErrQuitStakePool where
                     , "account! Make sure to withdraw your ", pretty rewards
                     , " lovelace first."
                     ]
+        ErrQuitStakePoolUnableToAssignInputs e ->
+            apiError err403 UnableToAssignInputOutput $ mconcat
+                [ "Unable to assign inputs from coin selection: "
+                , pretty e]
+        ErrQuitStakePoolUnableToAssignOutputs e ->
+            apiError err403 UnableToAssignInputOutput $ mconcat
+                [ "Unable to assign outputs from coin selection: "
```

Comment:
How is that a client error? This is a server error and it should not happen under any circumstances.